test/sourcemaps.js

```diff
@@ -141,7 +140,7 @@ suite('Source Maps', function() {
           '../fixtures/test-tree/some.js',
           '../fixtures/test-tree/text.txt',
           '../fixtures/test-tree/amd.js',
-          '../fixtures/test-tree/component.jsx',
+          '../fixtures/test-tree/unknown', // component.jsx
```

Comment:
Is this that sourcemaps issue you were talking about? I may be able to see if I can look into this too.